README.md

```diff
@@ -1,74 +1,177 @@
 # Tellor Data Specifications
 
-This repository contains data specifications for Query IDs. Query IDs are used as unique identifiers for data reported to and retrieved from the Tellor Oracle. In TellorX, `queryID` replaces the legacy `requestID` (tellor3 and earlier). Query ID is the common naming convention across this repository, the [oracle contracts](https://github.com/tellor-io/tellorX/blob/main/contracts/Oracle.sol) themselves, and the [Python telliot client](https://tellor-io.github.io/pytelliot/examples/legacy_query_example/).
+This repository contains the query specifications for interacting with the TellorX oracle.
 
-## Adding New ID's
+A Query specifies how to pose a question to the TellorX oracle, instructions for reporters on how to respond, including the format of the response, and any special dispute considerations.
 
-Make a PR in this repo to signal to reporters that the data is ready to reported and to show them how to assemble the corresponding queryID. 
+# Query Types
 
-### Fields:
+TellorX is designed to support arbitrary query types (`QueryType`).  A `QueryType` can have an arbitrary response type, specified by a structured
+ABI type string.  
 
-#### Question ID
+The following `QueryType`s are currently supported:
 
-    (name of file) can be guessed, but ultimately will be filled in by team.
+- LegacyRequest
+- TextQuery
+- CoinPrice (Under development)
 
-#### Query ID: 
+A `QueryType` can have multiple Query `parameters` that specify details of the query request (e.g. the token symbol for a `CoinPrice` query).
 
-    bytes32 query ID or structure for creating the ID (e.g. what to hash)
+Please submit a PR to this repository or contact the team if you would like to define a new `QueryType`. 
 
+# Query Descriptors
 
-#### Data Specifications
+Queries are uniquely identified by a special JSON string called the Query `Descriptor`.  In order to properly interact with the oracle, the `Descriptor` string must be provided *exactly* as defined by the [Query Descriptor Specification](#query-descriptor-specifications).
 
-    Technical Specifications of the value(s) requested
+The following is an example descriptor for a legacy query.  The `type` of query is a `LegacyRequest`.  The `LegacyRequest` query has a single integer parameter, called `legacy_id`.
```

Comment:
Why not call the integer parameter `requestId`, so it's more familiar?

Reply:
agree here, if you're on a legacy one, you know this terminology

Reply:
I guess I made a concerted effort to remove request_id from the entire codebase, so that it goes away from the TellorX lexicon.  That said, it could have been used in the LegacyRequest, which is the only thing that refers to the old tellor, but  not sure it's worth changing everywhere.